docs/readthedocs/source/doc/PythonAPI/LLM/optimize.rst

```diff
@@ -1,10 +1,24 @@
 BigDL-LLM PyTorch API
 =====================
 
-optimize model
+Optimize Model
 ----------------------------------------
 
-.. automodule:: bigdl.llm.optimize
+You can run any PyTorch model with one-line ``optimize_model`` to optimize, regardless of the library or API you are using.
```

Comment:
within one-line

Reply:
Maybe

You can run any PyTorch model with ``optimize_model`` through only one line of code change to benefit from BigDL-LLM optimization, regardless of the library or API you are using.